src/main/java/core/basesyntax/Pinguin.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Pinguin extends Animal {
```

Comment:
```suggestion
public class Penguin extends Animal {
```
Penguin should extend Bird and he should be able to swim 
https://github.com/mate-academy/jv-zoo
try to refactor all your classes of animals according to task description